.github/workflows/ci.yml

```diff
@@ -7,55 +7,71 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-latest
-          - macos-latest
-          - windows-latest
-        cmake_options:
-          - "-DWITH_OPENSSL=ON  -DWITH_GNUTLS=OFF -DWITH_GCRYPT=OFF" # build with OpenSSL
-          - "-DWITH_OPENSSL=OFF -DWITH_GNUTLS=ON  -DWITH_GCRYPT=ON"  # build with GnuTLS and Libgrypt
-          - "-DWITH_OPENSSL=OFF -DWITH_GNUTLS=OFF -DWITH_GCRYPT=OFF" # build without external encryption libraries
-          - "-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchain-cross-mingw32-linux.cmake" # crosscompile with MinGW toolchain
+          - ubuntu
+          - macos
+          - windows
+        # These are just the display names for the matrix, the actual options are generated in the 'include' section
+        cmake_options_name: ["OpenSSL", "GnuTLS and Libgcrypt", "No External Libraries", "MinGW Toolchain"]
```

Comment:
I would rather like to have it like with the verbatim cmake options as before for maintainability :-)

Reply:
Ok, I'll revert the shorter cmake options.
Do you also want the `disable_libs` verbatim?